app/controllers/idv/review_controller.rb

```diff
@@ -69,22 +76,14 @@ def submit_applicant
       resolution
     end
 
-    def idv_agent
-      @_agent ||= Proofer::Agent.new(
-        vendor: idv_vendor.pick,
-        kbv: FeatureManagement.proofing_requires_kbv?
-      )
-    end
-
-    def applicant_from_params
-      app_vars = idv_session.params.select { |key, _value| Proofer::Applicant.method_defined?(key) }
-      Proofer::Applicant.new(app_vars)
+    def password
+      params.require(:user)[:password]
+    rescue ActionController::ParameterMissing
+      ''
     end
 
-    def init_questions_and_profile(resolution)
-      idv_session.resolution = resolution
-      idv_session.question_number = 0
-      idv_session.profile_from_applicant(idv_session.applicant)
+    def invalid_password?
+      !current_user.valid_password?(password)
     end
   end
 end
```

Comment:
Since the user's password is used in order to encrypt their PII, we must collect that password at the moment we intend to encrypt it. Likewise, we use their password at log-in time to decrypt.